docs/data_sampling.md

```diff
@@ -26,8 +63,87 @@ After selecting MGRS tiles for each of these criteria, we removed duplicates.
 
 The following table summarizes the selection criteria for each class.
 
+| Class | Nr of Tiles | From highest |
+|---|---|---|
+Diversity | 400 | 2000
+Built-up | 300 | 300
+Built-up | 1000 | 1500
+Herbaceous wetland | 50 | 500
+Mangroves | 50 | 500
+Moss and lichen | 50 | 500
+Cropland | 800 | 3600
+Tree cover | 150 | 750
+Shrubland | 100 | 500
+Grassland | 200 | 500
+Bare / sparse vegetation | 50 | 500
+Snow and Ice | 25 | 500
+Permanent water bodies | 50 | 1000
+
+This resulted in a sample of 2728 MGRS tiles total in our sample. The resulting sample file can be downloaded from the following link
```

Comment:
Should we add some details on how we filtered these tiles based on cloud cover percentage?

Reply:
We actually did not use that at all for selecting the 2728 MGRS locations. The cloud filtering is done for each system when selecting specific scenes at each MGRS tile location. That is documented below.

Reply:
ak, okay!